### DISCUSSION TITLE
pnpm update changes "next" versions to absolute versions in package.json

### DISCUSSION THREAD

**Discussion by christophsturm** [pnpm/pnpm#7828] [Q&A]
I have a package that uses svelte 5 (next), so the package line in my package.json looks like this:
		`"svelte": "^5.0.0-next.84",`
when i run pnpm update and there is a newer version pnpm will change the line to `"svelte": "5.0.0-next.84",` , so the next time I want to update it will not update.
pnpm update --latest also does not work because "next" versions are not latest versions. 

what can I do to update my svelte package to latest without having to edit the package file afterwards?

**Comment by zkochan** [comment]
I guess it shouldn't remove the prefix in this case. However, I don't recommend using `^` with prerelease versions as the prerelease versions don't work reliably, they are just sorted alphabetically. I would recommend to update `svelte` like this:

```
pnpm update svelte@next
```

**Reply by christophsturm** [reply]
yes it would be great if it would not remove the ^. `pnpm update svelte@next` also removes it. if you point me where to change it I can try to make a pr. I now work in a monorepo with 5 projects using svelte next and after every update i have to manually edit 5 package.json files. I should probably automate it.
